tests/flow/test_multi_pattern.py

```diff
@@ -49,35 +49,35 @@ def populate_graph(cls):
 
     # Connect a single node to all other nodes.
     def test01_connect_node_to_rest(self):
-        global redis_graph
+        redis_graph
```

Comment:
This line can be safely deleted everywhere in this file, as variables are assumed to be global unless they are explicitly assigned within the function.